android/app/src/main/java/org/travlyn/ui/login/RegisterActivity.kt

```diff
@@ -0,0 +1,62 @@
+package org.travlyn.ui.login
+
+import android.content.Context
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import kotlinx.android.synthetic.main.activity_register.*
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+import org.travlyn.R
+import org.travlyn.api.UserApi
+import org.travlyn.api.model.User
+import org.travlyn.local.Application
+import org.travlyn.local.Formatter
+import org.travlyn.local.LocalStorage
+
+class RegisterActivity : AppCompatActivity(), Application {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_register)
+
+        val api = UserApi(application = this)
+
+        registerBtn.setOnClickListener {
```

Comment:
Do we want a minimum pw length or some additional stuff checked in frontend too?